spynnaker8/models/synapse_dynamics/synapse_dynamics_structural.py

```diff
@@ -0,0 +1,69 @@
+from spynnaker.pyNN.models.neuron.synapse_dynamics \
+    import SynapseDynamicsStructural as _BaseClass
+from spinn_front_end_common.utilities import globals_variables
+import numpy as np
+
+from spynnaker8.models.synapse_dynamics import SynapseDynamicsStatic
+
+
+class SynapseDynamicsStructural(_BaseClass):
+    __slots__ = [
+        "_f_rew",
+        "_p_rew",
+        "_weight",
+        "_delay",
+        "_s_max",
+        "_lateral_inhibition",
+        "_sigma_form_forward",
+        "_sigma_form_lateral",
+        "_p_form_forward",
+        "_p_form_lateral",
+        "_p_elim_dep",
+        "_p_elim_pot",
+        "_grid",
+        "_random_partner",
+        "_connections",
+        "_actual_row_max_length"]
+
+    def __init__(self, stdp_model=None, f_rew=10 ** 4, weight=0, delay=1,
```

Comment:
all these variables are magic to me, please create named vairables that explain what they do/are

Reply:
I don't know what this means. Where do I create these names variables?

Reply:
this is the slots, which i dont think you need to actually fill in. Plus the numebrs in the __init__. what does 10**4 gotta do with f_rew, what is it? 

on the same point where we dont do timeout=0.5 we do timeout=TIMEOUT_IN_SECONDS_FOR_BLAH